packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
   visRatio?: number | undefined;
   abscissaConfig: AxisConfig;
   ordinateConfig: AxisConfig;
-  interactionKeys?: InteractionKeys;
+  interactionKeys?: InteractionParams;
```

Comment:
What do you think of `interactions: Interactions` or `InteractionsConfig`?

Reply:
I would like to put `Interactions` but there is a `Interaction` interface for the interaction help button that is different. This being said, we can rename `Interaction` into `InteractionHelp` or something like that

Reply:
Renamed `InteractionParams` to `Interactions` and `Interaction` to `InteractionInfo`